src/modules/render_functions.js

```diff
@@ -1,30 +1,37 @@
-const appendLeaderboardUl = (name, score) => {
+import { loadLeaderBoard, postNewScore } from './api_manager.js';
+
+const populateLeaderBoard = (leaders) => {
   const ulLeaderboard = document.getElementById('leaderboard');
-  const liLeaders = document.createElement('li');
-  liLeaders.className = 'leaders';
-  liLeaders.textContent = `${name}: ${score}`;
-  ulLeaderboard.appendChild(liLeaders);
+  ulLeaderboard.innerHTML = '';
+  leaders.forEach((leader) => {
+    const liLeaders = document.createElement('li');
+    liLeaders.className = 'leaders';
+    liLeaders.textContent = `${leader.user}: ${leader.score}`;
+    ulLeaderboard.appendChild(liLeaders);
+  });
+};
+
+const formAddNewSubmit = (e) => {
+  e.preventDefault();
+  const scoreData = {
+    user: document.getElementById('username').value,
+    score: document.getElementById('score').value,
+  };
+  postNewScore(scoreData);
+  e.stopPropagation();
 };
```

Comment:
- [ ] [Optional] It would be better professionally if you clear the input fields after submitting a new score.